xml/Currency.xml

```diff
@@ -978,7 +978,7 @@
                 <narrative>Uzbekistan Sum</narrative>
             </name>
         </codelist-item>
-        <codelist-item status="withdrawn" withdrawal-date="2018-08-20">
+        <codelist-item status="withdrawn" withdrawal-date="2018-12-06">
```

Comment:
Guidelines on `withdrawal-date` are a bit unclear here, but… Intuitively, I’d expect this to reflect the actual withdrawal date, rather than the date it was changed in IATI.

Reply:
That is: I think 2018-08-20 made sense here. It has real-world meaning; it is cited elsewhere (e.g. [by wikipedia](https://en.wikipedia.org/wiki/Venezuelan_bol%C3%ADvar#Bol%C3%ADvar_soberano)).

Reply:
I was wondering this too, but the date 2018-08-20 doesn't match the ISO Currency amendments list....

Reply:
Might this not make it a bit harder to diagnose issues e.g. if you had a system that broke for VEF on a given date, but if the withdrawal date reported in the code-list change was a different date (one with real-world meaning), you might wrongly ignore that as the cause?

Reply:
> I was wondering this too, but the date 2018-08-20 doesn't match the ISO Currency amendments list....

@amy-silcock Could you provide a link to what you mean? I’m looking [at the document @rolfkleef linked](https://www.currency-iso.org/dam/downloads/dl_currency_iso_amendment_168.pdf) (screenshotted below).

![screen shot 2018-12-06 at 10 53 31](https://user-images.githubusercontent.com/464193/49579803-33af8280-f945-11e8-8e20-a38095bef8e5.png)

Reply:
> Might this not make it a bit harder to diagnose issues

@matmaxgeds git keeps track of timestamps of changes to replicated codelists. I think it’s actually a bad idea to duplicate that information inline, because it’s less well-defined what the date refers to (deployment date; merge date; real-world date).

Reply:
@andylolz I searched a bit around and found that in here https://www.currency-iso.org/en/home/amendments.html (excel table attached) the Amendment n. 168 for the Bolivar says `2018-08-02` rather than `2018-08-20` (a typo, I presume), so I think Amy just switched to when IATI amended its list to avoid ambiguity

Reply:
> the Amendment n. 168 for the Bolivar says 2018-08-02 rather than 2018-08-20 (a typo, I presume)

@samuele-mattiuzzo I don’t think it’s a typo. It’s **dated** 2 August 2018; it’s **effective from** 20 August 2018. That’s consistent with [the document Rolf posted](https://www.currency-iso.org/dam/downloads/dl_currency_iso_amendment_168.pdf) (screenshotted below)

![screen shot 2018-12-06 at 11 36 42](https://user-images.githubusercontent.com/464193/49582105-dbc84a00-f94b-11e8-8ef6-a963731f3ed2.png)

Reply:
I have no preference on which date goes into `activation-date` or `withdrawal-date`, but I read the maintenance guidelines as meaning "the date of change in the IATI codelist". Let's at least make it more explicit then.

Next time I'll add the activitation date, I wasn't aware of that :-)

Reply:
> but I read the maintenance guidelines as meaning "the date of change in the IATI codelist"

@rolfkleef We’re [talking about this](http://reference.iatistandard.org/203/codelists-guides/codelist-management/#code-status-addition-and-withdrawal-dates), right? If so, that’s not my reading of it (FWIW I think it’s ambiguous. Same for [the original proposal to add these attributes](https://support.iatistandard.org/hc/en-us/articles/214390846).)

> All new codes added to codelist from the release of version 2.02 (December 2015) will feature status="active" and activation-date attributes in the underlying GitHub source code repositories. Similarly, codes withdrawn from the release of version 2.02 (December 2015) will feature status="withdrawn" and withdrawal-date attributes.

To figure out when the IATI codelist was changed, you can use git (as you yourself [pointed out on the original proposal! :)](https://support.iatistandard.org/hc/en-us/articles/214390846/comments/204761986)) Since this is a replicated codelist, it seems to make more sense for the information captured to reflect the source codelist.

> Next time I'll add the activitation date, I wasn't aware of that :-)

These attributes are rarely used. Hence #244.

Reply:
@andylolz so yes we read this differently. Thanks for pointing back to that earlier discussion: the reason I don't really had an opinion about the date is because I'm not sure what use case it serves. I'll add that to #244.

I saw that for instance the withdrawal of code USS had been marked with 2018-08-20 as well: this is [the actual date of withdrawal from the IATI list](https://github.com/IATI/IATI-Codelists-NonEmbedded/commit/a157861a32efa68708c0c5a92e3b8620c4c31487), not the [withdrawal from the ISO list (2014-03)](https://www.currency-iso.org/dam/downloads/lists/list_three.xml).

So I agree @PetyaKangalova's change of the date is consistent with standing practice. If it gets resolved today still ;-)

Reply:
> So I agree @amy-silcock's change of the date is consistent with standing practice.

It’s consistent with that example, yes… I guess I wasn’t paying attention. But [the plan was to remove `USS` completely](https://github.com/IATI/IATI-Codelists-NonEmbedded/pull/268#discussion_r208595940), so I’m unsure how much value should be placed upon standing practice.

Anyway – you’ve convinced me that this is not a big issue. Happy with this change as it stands – don’t let me be the blocker here!